yacc_misplaced.py returned with 1

Traceback (most recent call last):
  File "/home/herczy/devel/ply/test/yacc_misplaced.py", line 64, in <module>
    yacc.yacc()
  File "/home/herczy/devel/ply/ply/yacc.py", line 3157, in yacc
    raise YaccError("Unable to build parser")
ply.yacc.YaccError: Unable to build parser

ERROR: /home/herczy/devel/ply/test/yacc_misplaced.py:32: Misplaced '|'
